ghost.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include "game_object.h"
+
+class Ghost : public GameObject {
+public:
+    Ghost(int x, int y);
+    void move(); 
+    void chasePacman(int pacmanX, int pacmanY); 
```

Comment:
The same:

```cpp
void chasePacman(const Point& pacmanPosition);

Reply:
You derived `Ghost` from `GameObject`, it is cool. But you need to re-define `update` method in all derived types.